Gemfile

```diff
@@ -43,7 +43,7 @@ end
 group :test do
   gem 'webmock', '~> 1.22'
   gem 'capybara'
-  gem 'capybara-webkit'
+  gem 'poltergeist'
```

Comment:
The JavaScript tests were falling over with Webkit, and I couldn't figure out why. Works fine with poltergeist - if you want me to switch it back can you give me a hand debugging webkit?